subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformValuesInjectionIntegrationTest.groovy

```diff
@@ -299,14 +299,14 @@ class ArtifactTransformValuesInjectionIntegrationTest extends AbstractDependency
                 }
             }
             
-            interface MakeGreenParameters extends TransformParameters {
-                @Input
-                String getExtension()
-                void setExtension(String value)
-            }
-            
             @CacheableTransform
-            abstract class MakeGreen implements TransformAction<MakeGreenParameters> {
+            abstract class MakeGreen implements TransformAction<Parameters> {
+                interface Parameters extends TransformParameters {
+                    @Input
+                    String getExtension()
+                    void setExtension(String value)
+                }
+            
```

Comment:
Do we have tests where the user defines the implementation for the getter/setter methods for a property?

Reply:
I don't think this is allowed. Those tests are part of the general decoration/injection infrastructure.